var/spack/repos/builtin/packages/sionlib/package.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+class Sionlib(AutotoolsPackage):
+    """Scalable I/O library for parallel access to task-local files"""
+
+    homepage = "https://www.fz-juelich.de/ias/jsc/EN/Expertise/Support/Software/SIONlib/_node.html"
+    url      = "http://apps.fz-juelich.de/jsc/sionlib/download.php?version=1.7.6"
+
+    maintainers = ['pramodk']
+
+    version('1.7.6', sha256='e85253ed3cd17a3b1c124ccd704caea3ad3c200abfcca9cc0851cb14f5a60691', extension='tar.gz')
+
+    variant('fortran', default=False, description='Enable Fortran support')
+
+    depends_on('mpi')
+    patch('for_aarch64.patch', when='target=aarch64:')
+
+    def configure_args(self):
+        args = []
+        spec = self.spec
+
+        if spec.satisfies('^intel-mpi'):
+            args.append('--mpi=intel2')
+        elif spec.satisfies('^mpich') or spec.satisfies('^mvapich2') or spec.satisfies('^hpe-mpi'):
```

Comment:
```suggestion
        elif spec.satisfies('^mpich') or spec.satisfies('^mvapich2') \
                or spec.satisfies('^hpe-mpi'):
```
Or something similar?